src/Views/SuccessView.vala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 elementary, Inc. (https://elementary.io)
+ * Copyright 2021 elementary, Inc. (https://elementary.io)
```

Comment:
Copyrights should include a date range instead of just the latest date.

```suggestion
 * Copyright 2019–2021 elementary, Inc. (https://elementary.io)
```